tools/wptrunner/docs/commands.rst

```diff
@@ -61,21 +61,26 @@ additional fields. All paths are relative to the commands.json.
   running the subcommand. :code:`virtualenv` must be true when this field is
   set.
 
-:code:`optional_requirements`
-  Similar to :code:`requirements`, but requirements specified in this list are
-  installed only when the corresponding command line flag is specified for the
-  subcommand. For example, given the following commands.json::
+:code:`conditional_requirements`
+  A key-value object. Values in this object are lists of paths similar to
```

Comment:
[optional] How about the following?

A key-value object. Each key represents a condition, and value represents additional requirements when the condition is met. The requirements have the same format as :code:`requirements`. Currently "commandline_flag" is...

Reply:
That's much nicer! Done.